app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -114,6 +120,14 @@ fun ReviewScreen(
           Button(
               onClick = {
                 Toast.makeText(context, "Review Added!", Toast.LENGTH_SHORT).show()
+                reviewViewModel.addReview(
+                    Review(
+                        owner = "default",
+                        text = textValue,
+                        parking = selectedParking.uid,
+                        rating = sliderValue.toDouble(),
```

Comment:
When doing a manual test i have 0.4997 instead of 0.5 which is probably caused by the cast to double. Can you double check ?

Reply:
Good catch, a new commit addresses this (reviews are now doubles to to 2 decimal places)

Reply:
I think I still have the problem.

Reply:
Apparently the problem is also present on the main so this will need a bugfix of it's own and it does not exist for all devices. I will approve your PR so that you can proceed to the 2nd one but we will need to fix this next sprint.